src/RayMarchedAtlasVolume.ts

```diff
@@ -56,16 +61,19 @@ export default class RayMarchedAtlasVolume {
 
     this.boxHelper = new Box3Helper(
       new Box3(new Vector3(-0.5, -0.5, -0.5), new Vector3(0.5, 0.5, 0.5)),
-      new Color(0xffff00)
+      BOUNDING_BOX_DEFAULT_COLOR
     );
     this.boxHelper.updateMatrixWorld();
     this.boxHelper.visible = false;
 
+    this.tickMarksMesh = this.createTickMarks();
+    this.tickMarksMesh.updateMatrixWorld();
+    this.tickMarksMesh.visible = false;
+
     this.cubeTransformNode = new Group();
     this.cubeTransformNode.name = "VolumeContainerNode";
 
-    this.cubeTransformNode.add(this.boxHelper);
-    this.cubeTransformNode.add(this.cubeMesh);
+    this.cubeTransformNode.add(this.boxHelper, this.cubeMesh, this.tickMarksMesh);
```

Comment:
Nitpick, but logically I feel like this order should be boxhelper, then tickmarks, then cubemesh.. just having boxhelper next to tickmarks...